lib/helper.js

```diff
@@ -1,25 +1,25 @@
 const Api = require('./api.js')
 
-var helper = function (client) {
+const helper = function (client) {
   this.client = client
 
   this.api = new Api(client)
 }
 
 helper.prototype.getAllSites = async function() {
-  response = await this.api.sites.get({ limit: this.client.PER_PAGE })
+  let sites = []
 
-  sites = response.sites
```

Comment:
Here we need to initiate sites with the response from the first page. Right now if all the sites are on the first page, return sites is empty I think.

Reply:
ah sorry I missed that, will fix